CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
   Fix travis cache directories
 * [#110](https://github.com/kowainik/summoner/issues/110):
   Print executing command
+* [#91](https://github.com/kowainik/summoner/issues/91):
+  Add warning fields for ghc
```

Comment:
@willbasky Could you also please update documentation in `README` and tell there that we have `warnings` field in configuration?